### PR TITLE
fix(usage plan): edits re: APOF

### PR DESCRIPTION
**Pull request by zuluecho9**
liaison work related to updating license docs with fact that apof is deprecated.

### CODE REVIEW

File: src/content/docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions.mdx [hunk 1]
```diff
@@ -66,7 +66,7 @@ These FAQs are meant to serve as a supplement to the main [New Relic pricing doc
     id="FAQ_6"
     title="Q: Can Event extended retention, additional Synthetics Checks, or additional New Relic Edge data be added to my account?"
   >
-    **A:** For additional retention for Events or Logs beyond the Standard Data Retention, contact your New Relic account executive. Note: Minimum requirement for Extended Retention requires an Annual Pool of Funds subscription.
+    **A:** For additional retention for Events or Logs beyond the Standard Data Retention, contact your New Relic account executive. Note: Minimum requirement for Extended Retention requires a [contract](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#usage-plans).
```
Comment (paperclypse) [inline src/content/docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions.mdx:69]: It feels weird to have a single word link like this.

maybe something more like: "requires a [Volume, Savings, or other contract]"

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx [hunk 2]
```diff
@@ -78,7 +78,7 @@ To learn how billable users are calculated, see [User count calculation](/docs/a
...
-* **Contracts**: Contract options include our **annual pool of funds** option, and other options. With contracts, an estimate of long-term expected usage (ingested data and billable users) is created and there's an agreed upon payment commitment. If an organization's usage exceeds that commitment, the excess usage is billed at the previously agreed-upon rates. These are available primarily to Pro and Enterprise edition organizations. 
+* **Contracts**: Contract options include our Volume Plan, Savings Plan, and other options. With contracts, an estimate of long-term expected usage (ingested data and billable users) is created and there's an agreed upon payment commitment. If an organization's usage exceeds that commitment, the excess usage is billed at the previously agreed-upon rates. These are available primarily to Pro and Enterprise edition organizations.
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:81]: I assume these have to be capitalized for legal reasons? Feels weird tho

File: src/content/docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions.mdx [hunk 3]
```diff
@@ -78,9 +78,9 @@ These FAQs are meant to serve as a supplement to the main [New Relic pricing doc
 
   <Collapser
     id="FAQ_8"
-    title="Q: How does the 'Free Tier' impact my pay-as-you-go (PAYG) or Annual Pool of Funds (APOF) subscription?"
+    title="Q: How does the 'Free Tier' impact my pay-as-you-go (PAYG) or contract subscription?"
   >
-    **A:** The Free Tier usage will be deducted automatically from the Monthly Product Usage for Pay-as-you-go or Annual pool of funds subscriptions.
+    **A:** The Free Tier usage will be deducted automatically from the Monthly Product Usage for Pay As You Go or [contract](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#usage-plans) subscriptions.
```
Comment (paperclypse) [inline src/content/docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions.mdx:83]: Seems like these an inconsistency between "pay-as-you-go" and "Pay As You Go" and I feel like we should pick one or the other.

Comment (paperclypse) [inline src/content/docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions.mdx:83]: Same comment as above re. single word links

File: src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/data-governance-baseline-ingest-guide.mdx [hunk 4]
```diff
@@ -287,9 +287,9 @@ For our [usage-based pricing model](/docs/accounts/accounts-billing/new-relic-on
...
+If you have a [contract](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#usage-plans), you'll likely have a monthly target budget for data ingest. For example, you may have set a target of 5TB per day and 100 full platform users. In this type of contract, users can be "traded off" but it's best to discuss this with other stakeholders in your organization to ensure you're getting the right mix for your observability goals. Although some customers will plan for variability in their consumption during the year, let's assume for now our monthly consumption budget is your total yearly contract billing amount divided by 12.
```
Comment (paperclypse) [inline src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/data-governance-baseline-ingest-guide.mdx:292]: same comment re. single word links

**Review by paperclypse — DISMISSED** [review]
Looks good to me. I left some minor comments.

File: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx:81]: yeah they capitalize a whole lot of stuff

**Pull request merged**